weasyl/commishinfo.py

```diff
@@ -82,7 +333,7 @@ def create_price(userid, price, currency="", settings=""):
 
     # Settings are at most one currency class, and optionally an 'a' to indicate an add-on price.
     # TODO: replace these character codes with an enum.
-    settings = "%s%s" % ("".join(i for i in currency if i in "epycmu")[:1],
+    settings = "%s%s" % ("".join(i for i in currency if i in "epycmufzn")[:1],
```

Comment:
Could you change these `"epycmufzn"` to use a constant built from `CURRENCY_CHARMAP`?